solutions/fully-configurable/DA-types.md

```diff
@@ -0,0 +1,49 @@
+## `metrics_filter`
+
+The `metrics_filter` variable allows you to control which custom metrics are collected by the IBM Cloud Monitoring agent. You can specify patterns to **include** or **exclude** certain metrics, giving you fine-grained control over the data sent to IBM Cloud Monitoring.
+
+### Type
+
+```hcl
+type = list(object({
+  include = optional(string)
+  exclude = optional(string)
+}))
+```
+
+### Description
+
+- **Purpose:**
+  Use `metrics_filter` to specify which metrics to include or exclude from collection. This is useful for reducing noise, focusing on relevant metrics, or optimizing resource usage.
+
+- **How it works:**
+  Each entry in the list can have an `include` or `exclude` field (or both).
+  - The value should be a string pattern (supports wildcards, e.g., `metricA.*`).
+  - `exclude` takes precedence over `include` if both match a metric.
+
+- **Default:**
+  ```hcl
+  default = []
+  ```
+  By default, no filtering is applied—all metrics are collected.
+
+### Example Usage
+
+```hcl
+metrics_filter = [
```

Comment:
the value should not contain the inout variable. Remove `metrics_filter = ` from the code snippet